app/page.tsx

```diff
@@ -40,6 +40,7 @@ export default function Home() {
   const [searchQuery, setSearchQuery] = useState('');
   const [loading, setLoading] = useState(true);
   const { data: session, status } = useSession();
+  const [descriptionDisplayMode, setDisplayMode] = useState<'showDescription' | 'hideDescription'>('showDescription');
```

Comment:
We don't really need to use `useState<>` in this case, since we are never changing the state of `descriptionDisplayMode`